src/shared/bus-unit-util.c

```diff
@@ -464,6 +464,29 @@ static int bus_append_cgroup_property(sd_bus_message *m, const char *field, cons
                 return 1;
         }
 
+        if (streq(field, "CPUQuotaPeriod")) {
+                usec_t u = 0;
+
+                if (!isempty(eq)) {
+                        r = parse_time(eq, &u, MSEC_PER_SEC);
```

Comment:
hmm, I am not sure about using ms as default unit here. Currently there's exactly one spot where we deviate from the default unit of seconds with otherwise universally use: the parsing of RLIMIT_RTTIME values. And the primary reason we parse with a default unit if µs there is for "downwards compat", i.e. simply because setting RLIMIT_RTTIME is widely supported in various tools (think: shell's ulimit command), where it always exposes the raw kernel unit of µs, and we should remain compatible to that. Now, i don't think CPUQuotaPeriod= is a similar case though, it's relatively exotic, and people seldom will echo these values directly to cgroupfs...

Or to say this differently: to deviate from the default unit of seconds for time values needs some major reasons, and I don't think this case qualifies. Having uniformity within the systemd codebase regarding time units is more important here...

Or in shorter words: please just use parse_sec() here!